core/kernel/persistence/smoothsql/search/driver/TaoSearchDriver.php

```diff
@@ -64,4 +64,17 @@ public function reserved($stringValue) {
         return $this->persistence->getPlatForm()->quoteIdentifier($stringValue);
     }
     
+    /**
+     * @inherit
+     */
+    public function random() {
+        $random = [
+            'mysql'      => 'RAND()', 
+            'postgresql' => 'random()', 
+            'mssql'      => 'NEWID()',
```

Comment:
I can't see a mssql driver change in PR https://github.com/oat-sa/lib-generis-search/pull/9. How is it working? Is mssql simply using the same driver as for mssql or postgresql?